### ISSUE TITLE
Cannot process argument transformation on parameter 'Body'. Cannot convert the "<h1>Hello</h1>" value of type      | "System.String" to type "Microsoft.Graph.PowerShell.Models.IMicrosoftGraphItemBody"

### ISSUE THREAD

**Issue by thenecroscope** [microsoftgraph/msgraph-sdk-powershell#339]
After upgrading from 0.7.1 to 0.9.1 my scripts are now failing when trying to send a simple HTML message.

Cannot process argument transformation on parameter 'Body'. Cannot convert the "Hello World" value of type
     | "System.String" to type "Microsoft.Graph.PowerShell.Models.IMicrosoftGraphItemBody"

Looks as though the parameters options have now changes from -BodyContent to -Body.

**Comment by thenecroscope** [comment]
The examples will probably need to be updated here also:
https://github.com/microsoftgraph/msgraph-sdk-powershell/blob/dev/samples/5-Teams.ps1

thanks again

**Comment by thenecroscope** [comment]
If I try this, I'm hitting the following error:

New-MgTeamChannelMessage -ChannelId $teamsConfigItemChannelID -TeamId $teamsConfigItemTeamID -Body @{content = "hello"}


Line |
12353 |          $scriptCmd = {& $wrappedCmd @PSBoundParameters}
      |                                      ~~~~~~~~~~~~~~~~~~
      | Cannot bind parameter 'Body'. Cannot convert the "Microsoft.Graph.PowerShell.Models.MicrosoftGraphItemBody"
      | value of type "Microsoft.Graph.PowerShell.Models.MicrosoftGraphItemBody" to type
      | "Microsoft.Graph.PowerShell.Models.IMicrosoftGraphItemBody".

**Comment by peombwa** [comment]
Thanks for reporting this bug. We've traced this issue to our code generator which expects `content` to start with uppercase `C` i.e. `Content`.

We are currently working on fix for this, and a workaround would to be ensure your property names are in PascalCase.
``` ps
New-MgTeamChannelMessage -ChannelId $ChannelId -TeamId $teamId -BodyParameter @{Body = @{Content = "hello"}} -Debug
```
or
``` ps
New-MgTeamChannelMessage -ChannelId $ChannelId -TeamId $teamId -Body @{Content = "hello"} -AdditionalProperties @{} -Debug
```

**Comment by thenecroscope** [comment]
Thanks Peter for the response. I'm still not having much luck using Pascal Case I'm afraid. I've copied both examples (thanks for the tip RE: debug and the alternative way of sending the message) but I'm now getting a new error.

I've tried PS5.1 and 7.1 P6, but they both are giving me the same error:

New-MgTeamChannelMessage_CreateExpanded1: C:\Users\xxxx\Documents\PowerShell\Modules\Microsoft.Graph.Teams.Team\0.9.1\exports\v1.0-beta\ProxyCmdletDefinitions.ps1:12353:23
 Line |
12353 |          $scriptCmd = {& $wrappedCmd @PSBoundParameters}
      |                        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      | An item with the same key has already been added. Key: content

thoughts?
thanks again

**Comment by jasonjoh** [comment]
@peombwa I reproduce what @thenecroscope is seeing as well using the two workaround commands you posted above. Let me know if I can help provide any further information.

**Comment by peombwa** [comment]
This was fixed with 1.0.1. `Install-Module Microsoft.Graph -AllowClobber -Force`.
